qt/widgets/common/src/FitPropertyBrowser.cpp

```diff
@@ -2079,40 +2079,39 @@ void FitPropertyBrowser::deleteTie() {
   // get name from composite function
   if (ci->property()->propertyName() == "Tie") {
 	  auto ties = h->getTies();
-	  QString QparName = ties.key(ci->property(), "");
-	  std::string parName = QparName.toStdString();
-	  QStringList fnNames;
-
-	  int iPar = -1;
+	  QString qParName = ties.key(ci->property(), "");
+	  std::string parName = qParName.toStdString();
+	  QStringList functionNames;
+	  // ithParameter = -1 => not found
```

Comment:
Consider converting this into an actual statement `constexpr auto notFound = -1`

Reply:
notFound could be confusing. notFound=1 would suggest that the statement is true (i.e. not found) and in reality it is the first parameter.

Reply:
I see that there is more to this than I thought.

One could make the case that:
- At the time when you set `ithParameter` to `notFound` you haven't yet found it, so the statement that you haven't found it is true anyway. 
- Or that at least naming it `notFound` is less confusing than `-1`.
- Or it could be called `sentinel` instead. 

But it's a minor issue which should be caught by #20654 regardless. So probably not worth holding up the PR for.